.github/workflows/quality-check.yaml

```diff
@@ -22,9 +22,18 @@ jobs:
     strategy:
       matrix:
         include:
+          - python-version: 3.6
+            tensorflow: "~=2.2"
+            tensorflow_probability: "~=0.10"
+          - python-version: 3.6
+            tensorflow: "~=2.3"
+            tensorflow_probability: "~=0.11"
           - python-version: 3.6
             tensorflow: "~=2.4"
             tensorflow_probability: "~=0.12"
+          - python-version: 3.6
+            tensorflow: "~=2.5"
+            tensorflow_probability: "~=0.12"
```

Comment:
0.13.0 is available as a release candidate. When it is released this can be changed.